st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -44,9 +44,19 @@ class SSHCommandTimeoutError(Exception):
     """
     Exception which is raised when an SSH command times out.
     """
-    def __init__(self, cmd, timeout):
```

Comment:
I will port those and other changes upstream to Libcloud since they would also come handy there. On top of that it also makes me unhappy that we need to copy those changes here instead of relying on upstream changes.